### PR TITLE
Add tests for atoms progress bar

### LINKED ISSUE #13: Backfill tests for Atoms/ProgressBar
Using other component tests as a guide, write automated tests to cover important scenarios in component. Try to avoid granular assertions of the markup. Stay as high-level as possible to avoid test fragility.

### PR DESCRIPTION
**Pull request by SaidShah**
- [x] Related GitHub issue(s) linked in PR description
- [x] Destination branch merged, built and tested with your changes
- [x] Code formatted and follows best practices and patterns
- [x] Code builds cleanly (no _additional_ warnings or errors)
- [x] Manually tested
- [x] Automated tests are passing
- [x] No _decreases_ in automated test coverage
- [x] Documentation updated (readme, docs, comments, etc.)
- [x] Localization: No hard-coded error messages in code files (_minimally_ in string constants)
- [x] New component and/ or properties? Make sure to update storybook

Closes #13

### CODE REVIEW

File: src/atoms/progress-bar/progress-bar.test.tsx [hunk 1]
```diff
@@ -1,5 +1,104 @@
...
+    ProgressBarStyles,
+    ProgressBarErrorClass,
+} from "./progress-bar";
+import { render, getByTestId } from "@testing-library/react";
 
 describe("ProgressBar", () => {
-    test.skip("TODO - https://github.com/AndcultureCode/AndcultureCode.JavaScript.React.Components/issues/13", () => {});
+    test("when default props, renders progress bar", () => {
+        // Arrange
+        const expected = "dataTestId";
```
Comment (brandongregoryscott) [inline src/atoms/progress-bar/progress-bar.test.tsx:13]: Usually we use the name `expected` when setting up a value that we're asserting on after the act step. Being as the act step doesn't calculate or modify anything related to this variable, and we're not actually asserting on its value, I would probably rename this to just `testId`

File: src/atoms/progress-bar/progress-bar.test.tsx [hunk 2]
```diff
@@ -1,5 +1,104 @@
...
+
+    test("when cssClassName set, renders with class name", () => {
+        // Arrange
+        const testCssClassName = "testCssClassName";
+
+        // Act
+        const { container } = render(
+            <ProgressBar
+                value={faker.random.number(100)}
+                cssClassName={testCssClassName}
```
Comment (brandongregoryscott) [inline src/atoms/progress-bar/progress-bar.test.tsx:32]: Nitpick - alphabetization

File: src/atoms/progress-bar/progress-bar.test.tsx [hunk 3]
```diff
@@ -1,5 +1,104 @@
...
+        expect(result).toHaveLength(1);
+    });
+
+    test.each`
+        style
+        ${ProgressBarStyles.Thick}
+        ${ProgressBarStyles.Thin}
+    `("when style prop set, renders with $style", ({ style }) => {
+        // Arrange
+        const expected = faker.random.number(100);
```
Comment (brandongregoryscott) [inline src/atoms/progress-bar/progress-bar.test.tsx:47]: Same as above - `expected` seems a bit misleading here

File: src/atoms/progress-bar/progress-bar.tsx [hunk 4]
```diff
@@ -21,6 +29,11 @@ interface ProgressBarProps {
     isErrored?: boolean;
     style?: ProgressBarStyles;
     value: number;
+
+    /**
+     * Unique identifier used to select the underlying <div> for functional/e2e testing
+     */
+    testId?: string;
```
Comment (brandongregoryscott) [inline src/atoms/progress-bar/progress-bar.tsx:36]: Nitpick - keep props alphabetical

**Review by brandongregoryscott — CHANGES_REQUESTED** [review]
Thanks for filling in these tests, @SaidShah! Left some feedback for naming & alphabetization

File: src/atoms/progress-bar/progress-bar.tsx (hunk 4 above)
Comment (SaidShah) [reply src/atoms/progress-bar/progress-bar.tsx:35]: @brandongregoryscott Thanks for the feedback. I have updated the PR if you want to take a second look.

**Review by brandongregoryscott — APPROVED** [review]

**Pull request merged**